packages/core/aqua-to-js-compiler/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "@fluencelabs/aqua-to-js-compiler",
+    "type": "module",
+    "version": "0.0.0",
+    "description": "Tool for generating aqua wrapper",
+    "main": "dist/index.js",
+    "scripts": {
+        "test": "vitest run",
+        "build": "tsc"
+    },
+    "keywords": [],
+    "author": "Fluence Labs",
+    "license": "Apache-2.0",
+    "dependencies": {
+        "@fluencelabs/aqua-api": "0.12.0",
```

Comment:
I no notice that the actual lib is not using aqua-api directly. It uses only types.
Maybe move it to dev dependencies and then this stuff won't be coupled as much because the format of compiler output shouldn't change that often. We will sometimes have two versions of aqua-api in cli but that's probably fine in my opinion
(we only have to make sure we add this lib to e2e in aqua repo)

Otherwise we would have to release new version of this lib every time aqua compiler changes and we would have to reexport aqua-api here

Reply:
Having to use 2 different version of aqua api is the thing i want to definitely avoid.

Reply:
Releasing new version of this lib after aqua-api version bump is fine. If your cli depends on it, same will happen with cli

Reply:
The idea is that you currenlty don't even use the aqua-api, so you are only dependeing on the types, you don't really depend on the implementation. That's why I think in this particullar case it could be fine if versions go out of sinc a bit once in a while, cause the types should change very rarely. I know that goind through the chain of CI for each release is a real pain that we live in every day. I don't know a lot of pain related to doing the same thing but more rarely because types will be changing very rarely

Also you will have to import and reexport functions that you don't even use at all in your lib, which is not ideal, I think you would agreee

Reply:
I moved this to devDeps, so no longer an issue